client/src/components/MentorItem.jsx

```diff
@@ -0,0 +1,53 @@
+import { React } from "react";
+
+import { Box, Grid, Typography, Link, Divider, Stack } from "@mui/material";
+
+import LinkedInIcon from "@mui/icons-material/LinkedIn";
+import LanguageIcon from "@mui/icons-material/Language";
+
+const MentorItem = ({ name, pronouns, linkedIn, websiteUrl, icon }) => {
+  return (
+    <Grid container direction="row" sx={{ margin: "15px" }}>
+      <Grid container item xs={4} alignItems="center">
+        <Box component="img" src={icon} sx={{ width: "90%" }} />
+      </Grid>
+      <Grid
+        container
+        item
+        xs={8}
+        sx={{ paddingLeft: "10px" }}
+        direction="column"
+      >
+        <Typography sx={{ fontWeight: 600 }}>{name}</Typography>
+        <Typography
+          sx={{ color: "#777F8C", fontWeight: 600, fontSize: "0.75rem" }}
+        >
```

Comment:
Got a lot of custom font styling here, is there any opportunity to dry up this code?

styled components or considering extension at the theme level?

Reply:
same comment as above related to jenna's theme reconfiguration